docs/hardware/5-virtual-device/2-quickstart/3-simulating-devices-coap.md

```diff
@@ -66,13 +92,9 @@ body: Hello
 
 #### With body from file (-f)
```

Comment:
Not sure where this file is supposed to live when I'm in the CLI. Is that something we should add in here?

Reply:
Do you mean change the way the file is referenced in the example? Right now it's perhaps a little confusing because the `./` included in `-f ./test.txt` is optional. We could change it to `-f test.txt` as a simplification.

Reply:
Thanks for discussing this with me on our call. I have added mention of the reference section for CLI and clarified file type/location in this commit: 6bc453e1bfbd06d4647360a39b22ab56bebcf58d